src/models/index.ts

```diff
@@ -21,6 +21,7 @@ import * as fieldToServerFull from './fields/field-to-server-full';
 import * as fileFromServer from './file-from-server';
 import * as fileToServer from './file-to-server';
 import * as grid from './grid';
+import * as imports from './imports';
```

Comment:
because `import` is a reserved word. So it was the easiest for now.
I don't know how to deal with this, I can:
- prefix (e.g. `$import`)
- rename, but to what ?

Reply:
Right! Maybe we should call it an `importRequest`, similar to `exportRequest`?

Reply:
Good!

I also notices that `exportRequest` was part of the settings route. But I figured that import was "detached" enough that I wanted to make a specific route for them.
Are we okay with this direction ?

Reply:
Yes! Export should have gotten its own route, but it's still there because of historical reasons. That triggers me: downloads are personal in the sense that you can only see and access the downloads you created yourself. I'm not sure that is also desired for uploads: it might be interesting to see what others are doing as well. In that case, a username (and hashId) would also be interesting to know..

Reply:
> downloads are personal in the sense that you can only see and access the downloads you created yourself. I'm not sure that is also desired for uploads: it might be interesting to see what others are doing as well.

I'm not sure for these files. Data imports, imo, are shared in to everyone in the environment. So I'd say here, no restriction.

But adding the username/hashid is still valuable if we want to restrict later, and also to know would did the import in the first place.
So I think it's valuable to add this as `createdBy: { name, hashid }` ?

Reply:
I agree! The userHashId can go in the importRequest model but the userName has to be a property of the controller responses (as it comes from a different table in the database)